### DISCUSSION TITLE
do you have info about how to add a custom gene Set for single sample scoring?

### DISCUSSION THREAD

**Discussion by ildelo1985** [biolab/orange3-bioinformatics#333] [Q&A] [answered]
do you have info about how to add a custom gene Set for single sample scoring?

**Comment by JakaKokosar** [comment, marked as answer]
@ildelo1985, of course, first, you must connect your data to the gene set widget via a 'custom gene set' input signal. I hope the image below explains the workflow. If you have any problems, please let me know.

![Screenshot 2023-04-03 at 09 48 57](https://user-images.githubusercontent.com/15876321/229446431-9a935375-34c2-43bc-a9a0-8f3d53517253.png)
